### ISSUE TITLE
Skip to main content link 

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#1703]
# Description

Let's add a "Skip to main content" link so that screen-reader or keyboard navigation users can skip the navigation menus.

We'll want to create a `SkipToContent` React component. We'll want to make sure it skips over to the correct content container for different user states.

We'll want to apply IDs to the elements that use the following classes. For example, we can use `#mainContent`.

* User signed in: `.dashboardSection`
* User not signed in: `.staticContent`

The component should look something like this:

```
<SKipToContent id="mainContent" />
```

The component itself should be an anchor link to `#mainContent`:

```
<a href="#maincontent">Skip to main content</a>
```

The component should only appear when a user tabs into a page. [Deque](https://www.deque.com/) has a good example of what we want to create.

# Screenshots

This is just a rough mockup of what we want this component to look like! Feel free to change the design as you see fit.

![image](https://user-images.githubusercontent.com/3010728/77232890-e7e81300-6b60-11ea-94d6-37247f071b4f.png)

![image](https://user-images.githubusercontent.com/3010728/77232900-f46c6b80-6b60-11ea-9e72-d8e1ef12545c.png)

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!

**Comment by discombobulateme** [comment]
I would like very much to work on it :)

**Comment by julianguyen** [comment]
Assigned you! For future reference, you can assign yourself to issues :)

Feel free to DM me on Slack if you have further questions about this issue!

**Comment by Victoria-Hodder** [comment]
As we discussed on Slack, @discombobulateme  and I will pair programme on this :) Looking forward to it!!

**Comment by discombobulateme** [comment]
Dear community, just a quick note to use here as a progression tracker. This is an image of the pre-approved design (at #design slack channel). We are still working on the component... studying a bit about react \o/ 

![bueLinkTop](https://user-images.githubusercontent.com/7980624/78684105-60d0c400-78f0-11ea-80b1-0f2c0192969a.png)
![blueSquare2](https://user-images.githubusercontent.com/7980624/78684139-68906880-78f0-11ea-839b-442cff9ddf4f.png)
![blueSquare3](https://user-images.githubusercontent.com/7980624/78684154-6d551c80-78f0-11ea-8d9a-76d62034826d.png)

**Comment by julianguyen** [comment]
Great work so far everyone!!!
